gobi

```diff
@@ -5,12 +5,24 @@ PLATFORMS=(iPhoneSimulator iPhoneOS)
 SOURCES=""
 PRODUCTS=""
 mkdir -p build/tmp
+mkdir -p _build/tmp
```

Comment:
I wanted to keep everything in one temporary directory, what are the downsides/problems with this approach here?